### ISSUE TITLE
When using an incompatible version of Chrome/Chromium, the renderer can hang

### ISSUE THREAD

**Issue by aviflax** [FundingCircle/fc4-framework#91]
For example, we’re currently using Puppeteer 1.4.x which is locked to Chromium 68, because Chromium 68 is the most recent version available on Alpine Linux, which we use for testing. I had Chromium 72 installed, and with that version, the renderer just stalled.

This is a bad experience…

Some ideas:

* Have some kind of timeout that kills the process if it never exits (and then fails)
* When the Clojure program invokes the Node renderer, it should pipe stderr of the renderer process to its own stderr, so users will see output incrementally
   * Not really a solution, but it would be nice to see what’s going on, and _exactly_ where it gets stuck.

**Comment by aviflax** [comment]
From a debugging session:

```
$ cat test/data/structurizr/express/diagram_valid_messy.yaml | time fc4-render --verbose | open -f -a /Applications/Preview.app
launching browser...
loading Structurizr Express from https://structurizr.com/express...
setting YAML and updating diagram...
(node:61368) UnhandledPromiseRejectionWarning: Error: Evaluation failed: SyntaxError: Unexpected identifier
    at ExecutionContext.evaluateHandle (/snapshot/renderer/node_modules/puppeteer/lib/ExecutionContext.js:106:13)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:61368) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:61368) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by aviflax** [comment]
I believe this was fixed by #106. We can always reopen if it’s observed again.
